### PR TITLE
Update metric-query-error-messages.md

### PR DESCRIPTION
**Pull request by zjiawei3**
## Purpose of this pull request

This pull request (PR) replays https://github.com/SumoLogic/sumologic-documentation/pull/781 . Should only be merged once the feature is released.

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by kimsauce — APPROVED** [review]

File: docs/metrics/metrics-queries/metric-query-error-messages.md [hunk 1]
```diff
@@ -15,9 +15,9 @@ Sumo Logic imposes limits on the input data for a query and the data output by t
 
 *Input data* is the data that matches the selector, prior to aggregation. Sumo Logic evaluates the volume of input data in terms of the number of time series.
 
-For a single metrics query row, Sumo Logic limits the number of input time series to 1000 for non-aggregate queries, and 37000 for aggregate queries (queries that have an aggregate operator like avg and max).
+For a single metrics query row, Sumo Logic limits the number of input time series to 1000 for non-aggregate queries, and 37000 for aggregate queries (queries that have an aggregate operator like avg and max). Besides, the total number of datapoints scanned by a single row of query is limited at 500,000,000 (500M) raw datapoints, or equivalent size of compressed datapoints.
```
Comment (michaljmatusiak92) [inline docs/metrics/metrics-queries/metric-query-error-messages.md:18]: Besides, the total number of datapoints scanned by a single row of query is limited at 500,000,000 (500M) raw datapoints, or 50M rollup datapoints (Link: https://help.sumologic.com/docs/metrics/introduction/metric-quantization/#rollup-types ).

as we agreed in the meeting

Comment (MariePMcGarry) [reply docs/metrics/metrics-queries/metric-query-error-messages.md:18]: Please change "Besides" to "In addition"

File: docs/metrics/metrics-queries/metric-query-error-messages.md [hunk 2]
```diff
@@ -15,9 +15,9 @@ Sumo Logic imposes limits on the input data for a query and the data output by t
 
 *Input data* is the data that matches the selector, prior to aggregation. Sumo Logic evaluates the volume of input data in terms of the number of time series.
 
-For a single metrics query row, Sumo Logic limits the number of input time series to 1000 for non-aggregate queries, and 37000 for aggregate queries (queries that have an aggregate operator like avg and max).
+For a single metrics query row, Sumo Logic limits the number of input time series to 1000 for non-aggregate queries, and 37000 for aggregate queries (queries that have an aggregate operator like avg and max). In addition, the total number of datapoints scanned by a single row of query limited at 500,000,000 (500M) raw datapoints, or 50M [rollup datapoints](https://help.sumologic.com/docs/metrics/introduction/metric-quantization/#rollup-types ).
```
Comment (betenkowski) [inline docs/metrics/metrics-queries/metric-query-error-messages.md:18]: 1. Inconsistent formatting: we mention `500,000,000 (500M)` raw DPs but only `50M` rollup DPs.
2. Unnecessary space between `#rollup-types` and `)`.

File: docs/metrics/metrics-queries/metric-query-error-messages.md [hunk 3]
```diff
@@ -15,9 +15,9 @@ Sumo Logic imposes limits on the input data for a query and the data output by t
...
 
-When a single row of a query scans more than 37000 time series, Sumo will stop scanning more data, and aggregate the results based on the scanned inputs. A message like this appears when the input limit is reached
+When a single row of a query scans more than 37000 time series or more than 500M datapoints, Sumo will stop after scanning the current time series, and aggregate the results based on the scanned inputs. A message like this appears when the input limit is reached:
```
Comment (betenkowski) [inline docs/metrics/metrics-queries/metric-query-error-messages.md:20]: *or more than 50M rollup datapoints*.

**Review by betenkowski — COMMENTED** [review]
I'd suggest splitting long lines, it will make the review process easier and it doesn't matter for markdown.

File: docs/metrics/metrics-queries/metric-query-error-messages.md (hunk 2 above)
Comment (betenkowski) [reply docs/metrics/metrics-queries/metric-query-error-messages.md:18]: Also, in the referenced document, we write *data points* not *datapoints*.

Comment (zjiawei3) [reply docs/metrics/metrics-queries/metric-query-error-messages.md]: I have made the update. Thanks!

File: docs/metrics/metrics-queries/metric-query-error-messages.md (hunk 3 above)
Comment (zjiawei3) [reply docs/metrics/metrics-queries/metric-query-error-messages.md]: Updated.

File: docs/metrics/metrics-queries/metric-query-error-messages.md [hunk 4]
```diff
@@ -15,9 +15,9 @@ Sumo Logic imposes limits on the input data for a query and the data output by t
 
 *Input data* is the data that matches the selector, prior to aggregation. Sumo Logic evaluates the volume of input data in terms of the number of time series.
 
-For a single metrics query row, Sumo Logic limits the number of input time series to 1000 for non-aggregate queries, and 37000 for aggregate queries (queries that have an aggregate operator like avg and max).
+For a single metrics query row, Sumo Logic limits the number of input time series to 1000 for non-aggregate queries, and 37000 for aggregate queries (queries that have an aggregate operator like avg and max). In addition, the total number of data points scanned by a single row of query limited at 500,000,000 (500M) raw data points, or 50,000,000 (50M) [rollup data points](/docs/metrics/introduction/metric-quantization/#rollup-types).
```
Comment (kimsauce) [inline docs/metrics/metrics-queries/metric-query-error-messages.md:18]:
```suggestion
For a single metrics query row, Sumo Logic limits the number of input time series to 1000 for non-aggregate queries, and 37000 for aggregate queries (queries that have an aggregate operator like `avg` and `max`). In addition, the total number of data points scanned by a single row of query limited to 500,000,000 (500M) raw data points, or 50,000,000 (50M) [rollup data points](/docs/metrics/introduction/metric-quantization/#rollup-types).
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Minor changes!

**Review by kimsauce — APPROVED** [review]

**Review by michaljmatusiak92 — APPROVED** [review]

**Comment by kimsauce** [comment]
Hi @MariePMcGarry , double-checking, did you get the green light to merge this?  @zjiawei3 said should only be merged once the feature is released.

**Comment by zjiawei3** [comment]
Yes. We are booked to release today. https://sumologic.slack.com/archives/C0496ALUQ0K/p1669667185860949?thread_ts=1668004684.198269&cid=C0496ALUQ0K  Thanks @kimsauce !

**Comment by kimsauce** [comment]
> Yes. We are booked to release today. https://sumologic.slack.com/archives/C0496ALUQ0K/p1669667185860949?thread_ts=1668004684.198269&cid=C0496ALUQ0K Thanks @kimsauce !

sweet thank you! congrats on the release.

**Pull request merged**